### ISSUE TITLE
Upgrate to 1.0.1 breaks my app

### ISSUE THREAD

**Issue by awendt** [codeclimate/ruby-test-reporter#146]
I just upgraded from 0.6.0 to 1.0.1 in an app. Because the dependency on `simplecov` was turned into a dev dependency in #121, I now get this error:

```
$APP/vendor/$BLA/gems/codeclimate-test-reporter-1.0.1/lib/code_climate/test_reporter/formatter.rb:7:in `require': cannot load such file -- simplecov (LoadError)
```

because the [formatter still requires simplecov](https://github.com/codeclimate/ruby-test-reporter/blob/master/lib/code_climate/test_reporter/formatter.rb#L7) 😢

**Comment by mbusch** [comment]
You should [check the updated doc](https://github.com/codeclimate/ruby-test-reporter#installation) before filing an issue.

**Comment by awendt** [comment]
@mbusch Thanks for the quick response. Admittedly, I wasn't looking.

**Comment by dblandin** [comment]
Hey @awendt,

I think you're right about the `simplecov` dependency. We no longer inject behavior into the SimpleCov test coverage process, but we still rely upon SimpleCov within the reporter, so it should be promoted from a dev dependency to a gem dependency.

**Comment by maxjacobson** [comment]
@awendt just released 1.0.3 which includes simplecov as a dependency, which should prevent this `LoadError`. Thanks for creating the issue to let us know about it!
